src/prospectScraper/info/2020ScheduledGames.csv

```diff
@@ -0,0 +1,131 @@
+,School,Conference,has2020SheduledGames
+0,Air Force,MWC,1.0
```

Comment:
I'm a little confused by the last field.  What does 1.0 represent here?  It should probably be an int value, and should represent the number of scheduled games for the team this fall.

Here's the Air Force football schedule, as I see it:
![image](https://user-images.githubusercontent.com/876146/95668776-13067400-0b3e-11eb-887b-adb1625d3732.png)

The number of scheduled games I see here is 8, so that's the number I'd expect here.  If the team forfeits or cancels any of those games, the number would change, but it's probably going to stay 8.